oak-run/src/test/java/org/apache/jackrabbit/oak/run/DataStoreCommandTest.java

```diff
@@ -390,6 +423,95 @@ public void testConsistencyVerboseRootPathWithoutVerbose() throws Exception {
         assertFalse(true);
     }
 
+    @Test
+    public void testDumpRefWithVerboseRootPath() throws Exception {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/gh/oak-indexing/jackrabbit-oak/pullRequest?prid=5079914)